plugins/module_utils/dellemc_idrac.py

```diff
@@ -45,7 +45,11 @@ def __enter__(self):
         self.sdk.importPath()
         self.handle = self.sdk.get_driver(self.sdk.driver_enum.iDRAC, self.idrac_ip, self.creds, pOptions=self.pOp)
         if self.handle is None:
-            msg = "Could not find device driver for iDRAC with IP Address: {0}".format(self.idrac_ip)
+            msg = "Unable to create a driver to communicate with idrac %s. This most often indicates one of the " \
+                  "following:\n\n" \
+                  '1. A bad username/password\n' \
+                  '2. An incorrect IP address\n' \
+                  '3. A failure in a TLS/SSL handshake (maybe due to a proxy/firewall?)' % self.idrac_ip
```

Comment:
@grantcurell  - instead of TLS/SSL handshake - reword probably to `unreachable iDRAC  <idrac_ip> `

Reply:
Unreachable IP address is accounted for #2. I was asked to add the TLS/SSL handshake issue specifically because it has come up frequently for support

Reply:
I updated it to:

```python
            msg = "Unable to communicate with iDRAC %s. This most often indicates one of the " \
                  "following:\n\n" \
                  '1. A bad username/password\n' \
                  '2. The iDRAC IP address is unreachable\n' \
                  '3. A failure in a TLS/SSL handshake (proxies/firewalls are a common cause of this problem)' % self.idrac_ip                  ```

Reply:
Thank you, The content is fine, we will get it reviewed by our Infodev team and get back on it